spec/unit/target/pod_target_spec.rb

```diff
@@ -184,55 +184,6 @@ module Pod
       end
     end
 
-    describe 'swift version' do
```

Comment:
all of these tests moved to the `analyzer_spec.rb` above as they are no longer logic that belong to this class.

Reply:
no tests were lost for this change.